### PR TITLE
org.blackcoin.blackcoin-more

### PR DESCRIPTION
**Pull request by expatjedi**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by expatjedi** [comment]
bot, build org.blackcoin.blackcoin-more

File: org.blackcoin.blackcoin-more.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          "sha256": "e73c68fd4a35cc7d629f62ccfb265be5b3fa259b162d28c31d15cf6228a8a90a"
+        },
+        {
+          "type": "archive",
+          "only-arches": ["aarch64"],
+          "url": "https://github.com/expatjedi/blk-flatpak-data/releases/download/13.2.0/blackcoin-more-13.2.0-aarch64-linux-gnu.tar.gz",
+          "sha256": "9816fa0772068b55686dda6e1c0eda276942980bfa53ee47495fee6a001311e8"
+        }
+      ]
+    },
```
Comment (barthalion) [inline org.blackcoin.blackcoin-more.json:40]: Any reason it uses your personal build instead of building it from source?

**Comment by barthalion** [comment]
Could you explain why "I am an upstream contributor to the project." is marked as done, but "I own the domain used in the application ID" is not? Just trying to establish how upstream is exactly related to this submission.

File: org.blackcoin.blackcoin-more.json (hunk 1 above)
Comment (expatjedi) [reply org.blackcoin.blackcoin-more.json:40]: We already built the binary within our building envoriment so no need to do same task again. I'm sure your system works great but ours too.

> Could you explain why "I am an upstream contributor to the project." is marked as done, but "I own the domain used in the application ID" is not? Just trying to establish how upstream is exactly related to this submission.

I'm one of the developer at blackcoin but not the leading one. If you wish, you may contact lead devs and confirm my role on project.

**Comment by expatjedi** [comment]
> Could you explain why "I am an upstream contributor to the project." is marked as done, but "I own the domain used in the application ID" is not? Just trying to establish how upstream is exactly related to this submission.

I'm one of the developer at blackcoin but not the leading one. If you wish, you may contact lead devs and confirm my role on project.

**Pull request closed without merging**